src/test/java/study/step4/RoundTest.java

```diff
@@ -0,0 +1,53 @@
+package study.step4;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class RoundTest {
+
+    private List<Car> cars;
+
+    @BeforeEach
+    void setUp(){
+        cars = new ArrayList<>();
+        cars.add(new Car("자동차1"));
+        cars.add(new Car("자동차2"));
+        cars.add(new Car("자동차3"));
+        cars.add(new Car("자동차4"));
+        cars.add(new Car("자동차5"));
+    }
+
+    @Test
+    @DisplayName("차 이름 비어있을 경우 예외 발생하는지 테스트")
+    void carNamesValidateTest(){
+        String [] carNames = {"", " "};
+        assertThatThrownBy(() ->{
+            new Round(carNames);
+        }).isInstanceOf(IllegalArgumentException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> new Round(carNames)).isInstanceOf(IllegalArgumentException.class);
```
의미 없는 중괄호는 생략하는것이 좋을것 같아요